test/cmd/txsim/cli.go

```diff
@@ -63,7 +63,7 @@ defined sequences; recursive patterns between one or more accounts which will co
 transactions. You can use flags or environment variables (TXSIM_RPC, TXSIM_GRPC, TXSIM_SEED, 
 TXSIM_POLL, TXSIM_KEYPATH) to configure the client. The keyring provided should have at least one
 well funded account that can act as the master account. The command runs until all sequences error.`,
-		Example: "txsim --key-path /path/to/keyring --rpc-endpoints localhost:26657 --grpc-endpoints localhost:9090 --seed 1234 --poll-time 1s --blob 5",
```

Comment:
[To reviewers] txsim does not support `rpc--endpoints` option. 
Also, the `grpc-endpoints` flag should be singular.

Reply:
good catch! Previously it was `rpc-endpoints` and `grpc-endpoints`. Now it's `grpc-endpoint`.  See this recent change: https://github.com/celestiaorg/celestia-app/pull/2326